lib/shared/bucketing/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@devcycle/bucketing",
-  "version": "1.0.21-alpha.0",
+  "version": "1.0.21",
```

Comment:
do not change, this can stay as is with the alpha tag.. otherwise this will publish a new version of the React SDK with version `1.0.21`